### PR TITLE
Feature/favorite page

### PR DESCRIPTION
**Pull request by Valentin-Morette**

### CODE REVIEW

File: frontend/src/components/Fav.jsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    { img: logo, name: "Favori 5" },
+    { img: logo, name: "Favori 6" },
+    { img: logo, name: "Favori 7" },
+    { img: logo, name: "Favori 8" },
+    { img: logo, name: "Favori 9" },
+  ];
+
+  return (
+    <>
+      <h1 className="text-center text-3xl my-16">Favoris</h1>
```
Comment (rocambille) [inline frontend/src/components/Fav.jsx:18]: put the heading in the section (and so the fragment will be useless)

**Review by rocambille — CHANGES_REQUESTED** [review]

File: frontend/src/components/Fav.jsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+      <section className="grid grid-cols-3 gap-5 gap-y-10 mx-6 mb-24">
+        {allFav.map((favorite) => (
+          <div className="flex flex-col items-center gap-1">
+            <img src={favorite.img} alt="favori" />
+            <p>{favorite.name}</p>
+          </div>
```
[the remark this answers is not in the document]
Comment (nibor51) [reply frontend/src/components/Fav.jsx]: commit [6bf9b11](https://github.com/WildCodeSchool/2022-03-JS-Reims-project-3-shop-around-me/pull/34/commits/6bf9b118aace69f5e3cd3c7f29abeced0a0442ee)

File: frontend/src/components/Fav.jsx (hunk 1 above)
Comment (nibor51) [reply frontend/src/components/Fav.jsx]: commit [2a59c29](https://github.com/WildCodeSchool/2022-03-JS-Reims-project-3-shop-around-me/pull/34/commits/2a59c294f7b8edc919f3fc026488c2af54ad4c08)

File: frontend/src/components/Navbar.jsx [hunk 3]
```diff
@@ -31,7 +31,7 @@ function Navbar() {
       id: 4,
       icon: faHeart,
       // URL to be determined later
```
Comment (nibor51) [inline frontend/src/components/Navbar.jsx:33]: dead code

**Review by nibor51 — CHANGES_REQUESTED** [review]

**Comment by rocambille** [comment]
conflicts to resolve

**Pull request merged**